system/jlib/jtrace.cpp

```diff
@@ -1199,13 +1262,47 @@ class CChildSpan : public CSpan
         injectlocalParentSpan(localParentSpan);
     }
 
+   /* CChildSpan(const char * spanName, CSpan * parent, stat_type thresholdMs)
```

Comment:
Initially I wanted to simply avoid the childspan::endSpan from calling  CSpan::endSpan();